src/components/Molecules/Task/index.jsx

```diff
@@ -0,0 +1,83 @@
+import styled from "styled-components";
+import React, { useState } from "react";
+import Checkbox from "../../Atoms/Checkbox";
+import EditButton from "../../Atoms/EditButton";
+import Input from "../../Atoms/Input";
+import TEXT from "../../../variables/texts";
+import FONTFAMILY from "../../../variables/font_family";
+import COLOR from "../../../variables/color";
+
+export const Task = ({
+  onTaskNameChange,
+  onTaskComplete,
+  taskName = "",
+  defaultIsEditing = false,
+}) => {
+  const [isEditing, useIsEditing] = useState(defaultIsEditing);
```

Comment:
```suggestion
  const [isEditing, setIsEditing] = useState(defaultIsEditing);
```
useState の2つめの返り値は `set〇〇` で受け取るのが一般的なので、習いましょう！